arangod/Graph/AttributeWeightShortestPathFinder.cpp

```diff
@@ -195,6 +199,16 @@ void AttributeWeightShortestPathFinder::clear() {
   clearCandidates();
 }
 
+#ifdef USE_ENTERPRISE
+bool AttributeWeightShortestPathFinder::pathContainsOnlySatellites() const {
```

Comment:
This implementation is not path based.
The algorithm right now limits the search to at most one Value.
And excludes all alternatives.